docs/streams/developer-guide.html

```diff
@@ -1094,8 +1094,8 @@
     //
     // OR
     //
-    KStreamBuilder builder = ...;  // when using the Kafka Streams DSL
-    Topology topology = builder.topology();
+    StreamsBuilder builder = ...;  // when using the Kafka Streams DSL
```

Comment:
I did add something in the last PR: https://github.com/apache/kafka/blob/trunk/docs/streams/developer-guide.html#L207-L225

Not sufficient?